getcontext/tracing/_helpers.py

```diff
@@ -2,6 +2,7 @@
 import os
 
 
+CONTEXT_DOMAIN = "https://api.context.ai"
 CONTEXT_TRACE_ENDPOINT = "https://api.context.ai/api/v1/evaluations/traces"
```

Comment:
Let's make this dynamic from CONTEXT_DOMAIN

Ie: CONTEXT_TRACE_ENDPOINT = CONTEXT_DOMAIN + "/api/v1/evaluations/traces"